docker-compose.yml

```diff
@@ -1,24 +1,33 @@
 # docker-compose.yml
-version: '3'
+
+version: '4'
```

Comment:
This docker-compose file wouldn't work for the reason that doesn't exist docker-compose [4 versions](https://docs.docker.com/compose/intro/history/). Use `3` version of docker-compose and please check for future if your code works after the changes